scripts/use-relative-md-links.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * A Markdownlint Plugin that checks whether the relative link urls are file-relative instead of
+ * directory-relative to ensure that mkdocs validates the links.
+ */
+
+// All external urls should have a scheme, i.e. http://, https:// or mailto:
+const externalUrl = /^[a-z0-9+.-]+:/;
+// linking within the same file is just (#<anchor-name>)
+const anchor = /^#/;
+const markdownFile = /.md/;
+const svg = /.svg/;
+
+module.exports = {
+	names: ['NAUTOBOTMD001', 'nautobotmd.filerelativelinks'],
```

Comment:
Can you convert the tabs to spaces in this file?

Reply:
Sure, the example from markdownlint was in tabs, so I assumed tabs was a requirement.